src/vivarium/framework/state_machine.py

```diff
@@ -83,7 +83,8 @@ def _groupby_new_state(
 
     """
     output_map = {o: i for i, o in enumerate(outputs)}
-    groups = pd.Series(index).groupby([output_map[d] for d in decisions])
+    # Pass grouper as Series to avoid FutureWarning about passing list of length 1
+    groups = pd.Series(index).groupby(pd.Series([output_map[d] for d in decisions]))
```

Comment:
What's the future warning?  These are pretty gross fixes

Reply:
"In a future version of pandas, a length 1 tuple will be returned when iterating over a groupby with a grouper equal to a list of length 1. Don't supply a list with a single grouper to avoid this warning."